src/MeshKernelNET/Api/DisposableMesh2D.cs

```diff
@@ -152,12 +160,24 @@ public int NumNodes
             set { numNodes = value; }
         }
 
+        public int NumValidNodes
```

Comment:
NumValidNodes should be a read-only property, it can be computed from the valid nodes.